telemetry/jetbrains/src/test/resources/testResultOutput

```diff
@@ -32,12 +31,14 @@ enum class Result(
 
     DashSeparated("dash-separated"),
 
-    UnderscoreSeparated("underscore_separated");
+    UnderscoreSeparated("underscore_separated"),
+
+    Unknown("unknown");
```

Comment:
Assume we should follow this pattern with all languages?

Reply:
Probs? I havent looked at the others yet.

In JetBrains we failed when we added new lambda runtimes and forgot telemetry so I ant to fall back to unknown in that case

Reply:
TS seems to be `foo|bar` so it will fail to compile so dont think we need it there

Reply:
C# does not generate a static from method, so probably not needed there either?

Reply:
what was the failure in JetBrains? was it a build time thing? cos if so - maybe it should fail so we see it....

Reply:
Nah, we converted the Lambda Runtime enum value to this version though the value `([Telemetry]Runtime.from([LambdaRuntime]runtime.toString)`